find-cursor.c

```diff
@@ -149,7 +149,7 @@ void draw(
 	// Setup display and such
 	char *display_name = getenv("DISPLAY");
 	if (!display_name) {
-		fprintf(stderr, "%s: cannot connect to X server '%s'\n", name, display_name);
+		fprintf(stderr, "%s: cannot connect to X server\n", name);
```

Comment:
Yeah, this made no sense facepalm: Would be better to mention a more specific error you, such as `%s: DISPLAY not set`.